### ISSUE TITLE
Stark nightly build publish fails on weekends due to version already published

### ISSUE THREAD

**Issue by christophercr** [NationalBankBelgium/stark#357]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#got-a-question-or-problem
</code></pre>

## Current behavior

The process to publish Stark nightly builds fails due to:

```bash
===> release-publish.sh publish checks ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>=
Publishing from Travis
=============================================
Verifying if this build has been triggered for a tag
 Nightly build initiated by Travis cron job
Verifying that the NPM_TOKEN is available
<=== release-publish.sh publish checks <==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<

===> release-publish.sh publish ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==
Publishing all packages

===> release-publish.sh publishing: stark-build ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>=
  Package path: /home/travis/build/NationalBankBelgium/stark/dist/packages-dist/stark-build
  Publishing TGZ file: ./nationalbankbelgium-stark-build-10.0.0-alpha.2-c2122e0.tgz
  Publishing the nightly release (with tag next)
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! You cannot publish over the previously published versions: 10.0.0-alpha.2-c2122e0. : @nationalbankbelgium/stark-build
```

## Expected behavior

Nightly build should not try to publish the same version twice.

## Minimal reproduction of the problem with instructions

Cron process for Stark nightly builds on Travis CI

## Environment

<pre><code>
Angular version: 5.2.4
Stark version: 10.0.0-alpha

For Tooling issues:
- Node version: 8.11.1 
- Platform: Travis CI 

Others:

</code></pre>

**Comment by dsebastien** [comment]
@christophercr isn't this one solved?

**Comment by christophercr** [comment]
Mmm I don't think is solved, in fact it happened again this weekend :disappointed: 

https://travis-ci.org/NationalBankBelgium/stark/builds/390339096

```
=============================================
Stark release publish @ npm
=============================================
Trace mode enabled!
=============================================
 PROJECT_ROOT_DIR: /home/travis/build/NationalBankBelgium/stark
 ROOT_PACKAGES_DIR: /home/travis/build/NationalBankBelgium/stark/dist/packages-dist

travis_fold:start:release-publish.sh----publish--checks
travis_time:start:release-publish.sh----publish--checks
===> release-publish.sh publish checks ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>=
Publishing from Travis
=============================================
Verifying if this build has been triggered for a tag
 Nightly build initiated by Travis cron job
Verifying that the NPM_TOKEN is available
<=== release-publish.sh publish checks <==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<==<

travis_time:end:release-publish.sh----publish--checks:start=1528621459485793592,finish=1528621459495389234,duration=9595642
travis_fold:end:release-publish.sh----publish--checks

travis_fold:start:release-publish.sh----publish
travis_time:start:release-publish.sh----publish
===> release-publish.sh publish ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==
Publishing all packages

travis_fold:start:release-publish.sh----publishing----stark-build
travis_time:start:release-publish.sh----publishing----stark-build
===> release-publish.sh publishing: stark-build ==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>==>=
  Package path: /home/travis/build/NationalBankBelgium/stark/dist/packages-dist/stark-build
  Publishing TGZ file: ./nationalbankbelgium-stark-build-10.0.0-alpha.2-18ddc7a.tgz
  Publishing the nightly release (with tag next)
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! You cannot publish over the previously published versions: 10.0.0-alpha.2-18ddc7a. : @nationalbankbelgium/stark-build

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-06-10T09_04_21_279Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stark-srcs@10.0.0-alpha.2 release:publish: `bash ./release-publish.sh --trace`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the stark-srcs@10.0.0-alpha.2 release:publish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-06-10T09_04_21_395Z-debug.log

```
